torchchat/generate.py

```diff
@@ -535,7 +535,6 @@ def decode_n_tokens(
         attention_backend: SDPBackend = torch.nn.attention.SDPBackend.MATH,
         **sampling_kwargs,
     ):
-        new_tokens, new_probs = [], []
```

Comment:
Not really used. This function yields individual tokens and probabilities, not the arrays.